kong.conf.default

```diff
@@ -105,7 +115,9 @@
                          # which proxies traffic from your consumers to your
                          # backend services. This value accepts IPv4, IPv6, and
                          # hostnames.
+                         #
                          # Some suffixes can be specified for each pair:
+                         #
```

Comment:
In the bullet list below, we did not update `-` to `*`. On purpose? (Btw, they read better as-is, if we could avoid changing them to `*` in the first place...)